bioio_ome_zarr/reader.py

```diff
@@ -129,6 +130,26 @@ def resolution_levels(self) -> Tuple[int, ...]:
             )
         )
 
+    @property
+    def resolution_level_dims(self) -> Dict[int, Tuple[int, ...]]:
+        """
+        Returns
+        -------
+        resolution_level_dims: Dict[int, Tuple[int, ...]]
+            resolution level dictionary of shapes.
+        """
+        if self._resolution_level_dict is None:
+            initial_resoluiton_level = self.current_resolution_level
+            resolution_level_dict = {}
+
+            for level in self.resolution_levels:
+                self.set_resolution_level(level)
+                resolution_level_dict[level] = self.shape
+            self._resolution_level_dict = resolution_level_dict
+            self.set_resolution_level(initial_resoluiton_level)
+
+        return self._resolution_level_dict
```

Comment:
since nothing about this function is zarr-specific, why not move the whole implementation up into bioio.BioImage?  Also I don't know if there is really a good reason to cache the info but I guess it doesn't hurt.

Reply:
resolved in [28041e4](https://github.com/bioio-devs/bioio-ome-zarr/pull/23/commits/28041e4edae2aa3ceeab0a8af1748e81d7a0fe71)